### PR TITLE
Fixed data/metadata graph and added nxv to prefix mappings

### PR DESCRIPTION
**Pull request by umbreak**
Fixes https://github.com/BlueBrain/nexus/issues/2327
Fixes https://github.com/BlueBrain/nexus/issues/2321
FIxes https://github.com/BlueBrain/nexus/issues/2329

### CODE REVIEW

File: delta/plugins/elasticsearch/src/test/resources/routes/elasticsearch-view-list-response.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "@context" : [
     "https://bluebrain.github.io/nexus/contexts/metadata.json",
     "https://bluebrain.github.io/nexus/contexts/search.json",
-    "https://bluebrain.github.io/nexus/contexts/all-metadata.json"
+    "https://bluebrain.github.io/nexus/contexts/listings-metadata.json"
```
Comment (bogdanromanx) [inline delta/plugins/elasticsearch/src/test/resources/routes/elasticsearch-view-list-response.json:5]: can we call this `search-metadata.json` instead?

**Review by bogdanromanx — CHANGES_REQUESTED** [review]
The context rename needs to be propagated also to the integration tests.

**Review by wwajerowicz — DISMISSED** [review]

File: delta/plugins/elasticsearch/src/test/resources/routes/elasticsearch-view-list-response.json (hunk 1 above)
Comment (umbreak) [reply delta/plugins/elasticsearch/src/test/resources/routes/elasticsearch-view-list-response.json:5]: Is not a search, is for listings. Search is when using the endpoint `/v1/views/{org}/{proj}/{id}/_search`

Comment (bogdanromanx) [reply delta/plugins/elasticsearch/src/test/resources/routes/elasticsearch-view-list-response.json:5]: we already have the search context that is used for listings. I think it makes sense.

**Review by bogdanromanx — CHANGES_REQUESTED** [review]
The modifications to the Integration tests are still missing.

**Comment by umbreak** [comment]
I'm not working on the integration tests right now, neither I have the powerful computer enough to orchestrate all of that. I'd be happy to make the changes tot he integration tests when there is a way for me to run them locally without plenty of timeouts.

**Comment by bogdanromanx** [comment]
> I'm not working on the integration tests right now, neither I have the powerful computer enough to orchestrate all of that. I'd be happy to make the changes tot he integration tests when there is a way for me to run them locally without plenty of timeouts.

It's only a matter of doing a string replace for `all-metadata.json` -> `search-metadata.json`

**Review by bogdanromanx — APPROVED** [review]

**Pull request merged**